### PR TITLE
Add `blacklists` and `remove_blacklist` methods

### PR DESCRIPTION
**Pull request by tsujigiri**

### CODE REVIEW

File: spec/integration/soapy_cake/admin_spec.rb [hunk 1]
```diff
@@ -105,4 +105,46 @@
...
+      {
+        advertiser_id: 14407,
+        offer_id: 9738,
+        affiliate_id: 15623,
+        sub_id: '119960714'
+      }
+    end
+
+    it 'removes blacklists', :vcr do
+      expect(subject.blacklists(blacklist_params)).to be_present
```
Comment (dziemba) [inline spec/integration/soapy_cake/admin_spec.rb]: You probably need `.to_a` here as well...

```ruby
Enumerator.new {}.present?
# => true
```

File: spec/lib/soapy_cake/admin_spec.rb [hunk 2]
```diff
@@ -234,4 +234,11 @@
       end
     end
   end
+
+  describe '#remove_blacklist' do
```
Comment (dziemba) [inline spec/lib/soapy_cake/admin_spec.rb]: How about adding `it_behaves_like 'a cake admin method'`?

File: spec/integration/soapy_cake/admin_spec.rb (hunk 1 above)
Comment (tsujigiri) [inline spec/integration/soapy_cake/admin_spec.rb]: :+1:

File: spec/lib/soapy_cake/admin_spec.rb (hunk 2 above)
Comment (tsujigiri) [inline spec/lib/soapy_cake/admin_spec.rb]: Done.

**Comment by dziemba** [comment]
:mushroom: 

(And please squash before merging, commits like `Add to_a` not very helpful)

**Pull request merged**